backend/src/modules/tournament-tree/core/application/event/EnqueueMatchWhenTournamentMatchWasEnded.ts

```diff
@@ -15,7 +15,7 @@ export class EnqueueMatchWhenTournamentMatchWasEnded implements EventHandler<Tou
     if (tournamentTree) {
       const { treeWithFinishedMatch, matchToEnqueue } = tournamentTree?.finishMatchInTreeAndGetNextOne(matchNumber, winnerId);
       await this.repository.save(treeWithFinishedMatch);
-      if (matchToEnqueue) {
+      if (matchToEnqueue && matchToEnqueue.firstTeam && matchToEnqueue.secondTeam) {
```

Comment:
Jesli wierzysz, ze to zadziała :) Bo ja w tym drzewku się tak nie łapię.

Reply:
Powinno. Problem był tak naprawdę, gdy "przepisywaliśmy" pierwszego zwycięzcę poziom wyżej, wtedy nie był sprawdzany drugi team i przekazywany w ciemno ten mecz.